source/TR-181/middle_layer_src/wanmgr_rdkbus_apis.c

```diff
@@ -1874,6 +1919,16 @@ ANSC_STATUS Update_Interface_Status()
             publishActiveStatus = TRUE;
 #endif
         }
+        if(strcmp(pWanDmlData->CurrentActiveDNS,CurrentActiveDNS) != 0)
```

Comment:
Check for empty CurrentActiveDNS. Update_Current_ActiveDNSmay fail to open the dns conf and return empty string